lib/middleware.lib.ts

```diff
@@ -13,11 +14,13 @@ export function isRequestTokenAuthorized(
   res: Response,
   next: NextFunction,
 ) {
-  console.log('started auth');
+  // console.log('started auth');
   const authToken = req.header(config.get('authToken'));
   User.searchUserByAuthToken(authToken, function (result) {
     if (result.res == null) {
-      res.status(403).send(result.err ?? 'user is not authorised');
+      res
+        .status(StatusCodes.FORBIDDEN)
```

Comment:
If the user is unauthorised shouldn't a 401 unauthorised error be sent instead?

Reply:
in reality, the **401 Unauthorised** status should be called _unauthenticated_. It should be sent if:
- the client sent incorrect login details
- the server could not understand the authentication method
- the client did not send the authentication

**403 Forbidden** should be sent when the server understands the authentication but that user is not _authorised_ to access or change the content.

Reply:
I think this was the only way that I could get the test to pass? There's a function that checks the authToken and returns forbidden if it's wrong which is before the unauthorised return of the endpoint function. I don't know if this test even worked originally